client/src/components/login-reg-components/LandingCarousel.jsx

```diff
@@ -0,0 +1,55 @@
+import React from "react";
+import Carousel from "react-bootstrap/Carousel";
+import slide1 from "../../static/static-imgs/jacqueline-munguia-1pAwJiCD60c-unsplash.jpg";
+import slide2 from "../../static/static-imgs/pexels-juan-pablo-serrano-arenas-967016.jpg";
+import slide3 from "../../static/static-imgs/pexels-kat-smith-736842.jpg";
+import slide4 from "../../static/static-imgs/pexels-pixabay-208147.jpg";
+import slide5 from "../../static/static-imgs/pexels-pixabay-264196.jpg";
```

Comment:
What values do you get from console logging these imports?

Reply:
The value is the img filepath